iOS/GPUImage/nuget/GPUImage.nuspec

```diff
@@ -3,14 +3,14 @@
   <metadata>
     <id>GPUImage</id>
     <title>GPUImage</title>
-    <version>0.1.7.1</version>
+    <version>0.1.7</version>
```

Comment:
0.1.7.2

Reply:
Also need to change the packages version that is referenced to 0.1.7.2